pkg/app/handler.go

```diff
@@ -88,14 +89,27 @@ func (d *DidDocumentHandler) GetByENSDomain(w http.ResponseWriter, r *http.Reque
 
 func (d *DidDocumentHandler) GetGist(w http.ResponseWriter, r *http.Request) {
```

Comment:
I would totally remove this function, because we define how to retrieve a gist using standard API